src/presentation/pages/SignUp/styles.scss

```diff
@@ -30,6 +30,56 @@
 
       animation: SignUpAnimation 1.2s;
 
+      .signUpPasswordStrong{
+        height:5px;
+        width:300px;
+        border-radius: 5px;
+        background-color: $green;
+        margin-bottom: 20px;
+      }
+      .signUpPasswordStrongScript{
+        color: $green;
+        text-align: center;
+      }
+      .signUpPasswordMedium{
```

Comment:
you repeat yoursealf a lot of time here too, you can create only one class and add the color, because I see its the only thing that changes